### PR TITLE
[FIX] account_statement_ext: Catch also old exceptions (osv.except_osv)

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by rvalyi** [comment]
:+1:  LGTM

File: account_statement_ext/statement.py [hunk 1]
```diff
@@ -420,7 +420,7 @@ def button_confirm_bank(self, cr, uid, ids, context=None):
                     self.create_move_from_st_line(
                         cr, uid, st_line.id, company_currency_id,
                         st_line_number, context)
-                except orm.except_orm, exc:
+                except (orm.except_orm, osv.except_osv), exc:
```
Comment (bwrsandman) [inline account_statement_ext/statement.py]: Can you use the 2.7 syntax of `as exc` and not the deprecated `, exc`

**Comment by pedrobaeza** [comment]
Done!

**Comment by gurneyalex** [comment]
:+1:

**Comment by laetitia-gangloff** [comment]
Thank you !

:+1:

**Pull request merged**